### DISCUSSION TITLE
Cant Add components it says Invalid configuration in the components.json

### DISCUSSION THREAD

**Discussion by LugawPlain** [shadcn-ui/ui#4966] [Q&A]
![Untitled](https://github.com/user-attachments/assets/ee73bfbf-7a93-40d4-b85f-ead918d507c4)

This error shows up when i try to install components in the command line
So i opened a old project of mine and tried to install the card component i dont know why it says it 
i tried to delete the components and init it again but still says the same error i tried to make a new nextjs project and add components it seems fine on the other project but in this proj it wont work even if i reset the components folder i dont know why cant seem to find any solution for this

**Comment by LugawPlain** [comment]
nvm its just npx shadcn@latest add card
not npx shadcn-ui@latest add card

**Reply by BashmailOmar** [reply]
You saved my life

**Comment by potomantojoe** [comment]
I'm having the same challenge here. Anyone with solution?

**Reply by LugawPlain** [reply]
i typed not npx shadcn-ui@latest add card since i was following a youtube tutorial but i was supposed to be  npx shadcn@latest add card hope this helps

**Reply by potomantojoe** [reply]
Solved the problem, and I wasn't conscious of the fact you were pointing out though until now. It makes sense, I probably did the same thing (I just can't confirm that now since the issue has been taken out of the way). I did something else though and it worked. So yh, all that matter is problem solved, I guess :) The link to my issue and how I overcame it here https://github.com/shadcn-ui/ui/issues/5577

But hey, thanks for the feedback and contribution. I'm grateful.
